src/cascadia/TerminalSettingsModel/ApplicationState.cpp

```diff
@@ -60,38 +60,40 @@ using namespace ::Microsoft::Terminal::Settings::Model;
 
 namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 {
+    ApplicationState::ApplicationState(std::filesystem::path path) noexcept :
+        BaseApplicationState{ path } {}
+
     // Returns the application-global ApplicationState object.
     Microsoft::Terminal::Settings::Model::ApplicationState ApplicationState::SharedInstance()
     {
         static auto state = winrt::make_self<ApplicationState>(GetBaseSettingsPath() / stateFileName);
+        state->Reload();

```

Comment:
the problem this was fixing was that we were originally doing the `Reload` in the ctor, but if we did a Reload in the `BaseApplicationState` ctor, then there wasn't an implementation for `read` yet (or at least, there wasn't when I first wrote this).

I'm gonna massage all this in response to https://github.com/microsoft/terminal/pull/11222#discussion_r708847927, so this might all be moot